uiowa/uiowa
# trim trailing slashes from SiteNow request form URL value

fixes #3176

## To test

- checkout **main** and sync sitenow.uiowa.edu
- Fill out the form here with some different URLs (some including a trailing slash e.g. https://foo.uiowa.edu/): https://sitenow.local.drupal.uiowa.edu/request-site
- See the errors, see the problem.
- checkout this branch, pull, `drush @sitenow.local cr`, refresh the form and try the trailing slash URL again. It should pass.

Reroute email should prevent these emails from reaching its-web.
